CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [14.0.0]
+### Changed
+- **BREAKING** Update `@metamask/polling-controller` from `^8.0.0` to `^11.0.0` ([#448](https://github.com/MetaMask/smart-transactions-controller/pull/448))
```

Comment:
we usually use Bump instead of Update ?

Reply:
NVM, In this package I see that we are using Update instead

Reply:
Yea, I was in doubt actually because in this `CHANGELOG` we are using Update